src/custom/components/SideBanner/index.tsx

```diff
@@ -13,8 +13,12 @@ const HEIGHT = 440
 const ANNIVERSARY_TWEET_TEMPLATE =
   'Holy CoW, today @MEVprotection is 1 year old! Check out their UI growth over time to see the CoW-evolution https://youtu.be/nxU11DmBVMk'
 
+// create an enum with the types of banners we want to show
+export enum BannerType {
+  ANNIVERSARY = 'anniversary',
+}
 export interface BannerProps {
-  type: string
+  type: BannerType
```

Comment:
ah maybe my suggestion was overkill :D thought we had more types. anyways good to key. i'd probably just stick with a normal `type` in the BannerProps declaration like

```ts
export interface BannerProps {
  type: 'anniversary' | 'some_other' | 'another_another' 
  isVisible: boolean
}
```

but this is also fine

Reply:
Ah, cool good to know when to use it. I Intended for the`SideBanner` component to be re-usable for all types of banners. So perhaps an `enum` might be a good 'future' proof approach in this case?